lib/ngx/balancer_stream.lua

```diff
@@ -0,0 +1,82 @@
+-- Copyright (C) Yichun Zhang (agentzh)
```

Comment:
Also, I hope we can still use the `ngx.balancer` module in the stream context. We could do a simple dispatch to `ngx.balancer.stream` in the top-level scope of `ngx.balancer`.